doc(release) Release notes collect 24.10 (#4428)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Collect

### 24.10.7

> Release date: `June 19, 2025`

Reviewer: ```suggestion
Release date: `June 20, 2025`
```

<details>
  <summary>Bug fixes</summary>

> - [Broker] Acknowledgements now handled for all BA types in BAM.

Reviewer: ```suggestion
- [Broker] Acknowledgements are now handled for all types of BAs in BAM.
```

## Centreon Monitoring Agent

### 24.10.5

> Release date: `June 19, 2025`

Reviewer: ```suggestion
Release date: `June 20, 2025`
```

<details open>
  <summary>Enhancements</summary>

- [CMA] Added a native Eventlog check for Windows.
- [CMA] Added a native Process check for Windows.

> - [CMA] Added a No TLS connection mode (limited duration).

Reviewer: ```suggestion
- [CMA] Added a No TLS connection mode (this mode has a limited timeframe).
```

> - [CMA] Added a No TLS connection mode with a limited duration.

Reviewer: ```suggestion
- [CMA] Added a No TLS connection mode. This mode is only active within a limited timeframe.
```

> - [CMA] Added a No TLS connection mode (this mode has a limited timeframe).

Reviewer: ```suggestion
- [CMA] Added a No TLS connection mode with a limited duration.
```

## Centreon Gorgone

### 24.10.5

> Release date: `June 19, 2025`

Reviewer: ```suggestion
Release date: `June 20, 2025`
```

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Removed useless error present at every first start of gorgone.

Reviewer: ```suggestion
- [Gorgone] Removed an unnecessary error that was present at every first start of Gorgone.
```
